### ISSUE TITLE
Something's gone wrong | Opening extension settings

### ISSUE THREAD

**Issue by LautiLosio** [flexagoon/rounded-window-corners#3]
Hi, just found out about this fork. I'm really grateful for the work you've put into it.

I think I found this bug when trying to open the extension settings

![image](https://github.com/flexagoon/rounded-window-corners/assets/38726997/9311a904-7f52-4f97-8054-c8c824c1d0ce)

The settings of extension rounded-window-corners@fxgn had an error:
```
TypeError: this._rounded_in_maximized_switch is undefined

Stack trace:
  set cfg@file:///home/lau/.local/share/gnome-shell/extensions/rounded-window-corners@fxgn/preferences/widgets/rounded_corners_item.js:101:9
  build_ui@file:///home/lau/.local/share/gnome-shell/extensions/rounded-window-corners@fxgn/preferences/pages/general.js:95:33
  _init@file:///home/lau/.local/share/gnome-shell/extensions/rounded-window-corners@fxgn/preferences/pages/general.js:50:14
  @file:///home/lau/.local/share/gnome-shell/extensions/rounded-window-corners@fxgn/preferences/pages/general.js:33:4
  pages@file:///home/lau/.local/share/gnome-shell/extensions/rounded-window-corners@fxgn/preferences/index.js:9:17
  fillPreferencesWindow@file:///home/lau/.local/share/gnome-shell/extensions/rounded-window-corners@fxgn/prefs.js:28:28
  _loadPrefs@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:41:18
  async*_init@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:26:14
  ExtensionPrefsDialog@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:17:4
  OpenExtensionPrefsAsync@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:139:33
  async*_handleMethodCall@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:373:35
  _wrapJSObject/<@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:408:34
  _init/GLib.MainLoop.prototype.runAsync/</<@resource:///org/gnome/gjs/modules/core/overrides/GLib.js:266:34
  
```

**Comment by flexagoon** [comment]
I identified the cause of the issue:

https://github.com/flexagoon/rounded-window-corners/blob/c73139f10e9a931c8a92cdbd3a7c47ac2205dfc5/src/preferences/widgets/rounded_corners_item.ts#L32-L41

Those lines were previously removed by TypeScript and were not present in the built JS. After the last commit, they stay in the JS file, and overwrite the variables with the same names that are defined by GTK.

This is most likely because I upgraded from TS 4 to TS 5. I'll remove those lines later when I'll be at my computer.
